de.dlr.sc.virsat.model.concept/src/de/dlr/sc/virsat/model/concept/generator/tests/GenerateAllTests.xtend

```diff
@@ -189,11 +189,13 @@ class GenerateAllTests extends AGeneratorGapGenerator<EObject> {
 			listOfTests.add(it + "Test")
 		]
 		
+		importManager.register(concept.fullQualifiedName + ".validator." + concept.name.replace("de.dlr.sc.virsat.model.extension.", "") + "Validator" + "Test")

```

Comment:
What is this about? Maybe a comment in the code could help ehre a bit. I am confused by the replace statement. Nevertheless i guess there is a good reason for it.